deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
+SCRIPTPATH=$(pwd -P)
+FILE1="docker-compose-base.yaml"
+FILE2="docker-compose-disk.yaml"
+FILE3="disk.env"
+
+#Prints instructions to show possible commands
+instructions() {
+	echo
+	echo "Usage:"
+	echo " To start the GAIA Hub type: $0 start."
+	echo " To stop the GAIA Hub type: $0 stop."
+	echo " To check if GAIA Hub is running type: $0 status."
+	echo " Simply typing $0 displays this help message."
+	echo
+}
+
+#Checks files I need exist
+check_files_exist() {
+	# If a file I need is missing, inform the user.
+	if ! test -f "$FILE1"; then 
+		echo "Missing $FILE1. Did you delete it?"
+		return 0
+	fi
+	if ! test -f "$FILE2"; then
+		echo "Missing $FILE2. Did you delete it?"
+		return 0
+	fi
+	if ! test -f "$FILE3"; then
+		echo "Missing $FILE3. Looks like you forgot to create one."
+		return 0
+	fi
+	# If all files I need exist, then continue
+	return 1
+}
+
+#Checks if already running my containers
+check_containers() {
+	if [[ $(docker compose -f ${SCRIPTPATH}/${FILE1} -f ${SCRIPTPATH}/${FILE2} --env-file ${SCRIPTPATH}/${FILE3} ps -q) ]];
+	then
+		# docker running
+		return 0
+	fi
+	# docker not running
+	return 1
+}
+
+gh_status() {
+	if check_containers; then
+		echo "GAIA HUB running."
+		return 1
+	fi
+	if ! check_containers; then
```

Comment:
Shouldn't this just be an `else` instead of an additional `if` statement?

Reply:
Ok. Changed.